### ISSUE TITLE
job queue no longer shows running or queued jobs

### ISSUE THREAD

**Issue by ollyg** [netdisco/netdisco#601]
As reported by _KirikkH_ on IRC.

**Comment by ollyg** [comment]
> I have 2 installations, one running 2.40.3 and this new installation running 2.42.9. on 2.40.3 I do get running and queued jobs

**Comment by inphobia** [comment]
seems something in github closed this when you write fix #issuennumber in your online commit.

but this should fix it,  will need new netdisco release asap. will update Changes now so @ollyg just needs to hit the release button

**Comment by inphobia** [comment]
hm, that wasn't this issue. will check between working 006 & not working 009 & reoport back

**Comment by inphobia** [comment]
ffs don't keep closing this issue

**Comment by inphobia** [comment]
seems 5452eaa might be the issue, either reverting the changes at line 273 or changing them to
``` perl
sub jq_log {
  return schema('netdisco')->resultset('Admin')->search({}, {
    log      => { '-not_like' => 'duplicate of %' },
    prefetch => 'target',
    order_by => { -desc => [qw/entered device action/] },
    rows     => (setting('jobs_qdepth') || 50),
  })->with_times->hri->all;
}
```
seems to fix the issue, at least for me.

dunno how, dunno why. no dbix guru, not in the least....

**Comment by ollyg** [comment]
thanks @inphobia !
easy and quick fix I hope, just release 2.042010 to CPAN
